.github/dependabot.yml

```
@@ -4,11 +4,11 @@ updates:
     schedule:
-      interval: daily
-      timezone: Asia/Seoul
+      interval: "daily"
+      timezone: "Asia/Seoul"
     open-pull-requests-limit: 5
     allow:
       - dependency-name: "@team-frieeren/*"
     ignore:
       - dependency-name: "@team-frieeren/*"
-        update-types: ["version-update:semver-major"]
+        update-types: ["version-update:semver-major"]
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)